gvm/transform/object/classes.py

```diff
@@ -0,0 +1,1165 @@
+        if root is None:
+            return None
+        uuid = root.get("id")
+        owner = Owner.resolve_owner(root.find("owner"))
+        name = get_text_from_element(root, "name")
+        comment = get_text_from_element(root, "comment")
+
+        creation_time = get_datetime_from_element(root, "creation_time")
+        modification_time = get_datetime_from_element(root, "modification_time")
+
+        writable = get_bool_from_element(root, "writable")
+        in_use = get_bool_from_element(root, "in_use")
+
+        permissions = Permission.resolve_permissions(root.find("permissions"))
+        port_count = PortCount.resolve_port_count(root.find("port_count"))
+        port_ranges = PortRange.resolve_port_ranges(root.find("port_ranges"))
+
+        port_list = PortList(
+            uuid,
+            owner,
+            name,
+            comment,
+            creation_time,
+            modification_time,
+            writable,
+            in_use,
+            permissions,
+            port_count,
+            port_ranges,
+        )
+        return port_list
+
+    @staticmethod
+    def resolve_port_lists(root: etree.Element):
+        result_list = []
+
+        for child in root:
+            if child.tag == "port_list":
+                port_list = PortList.resolve_port_list(child)
+                result_list.append(port_list)
+
+        if len(result_list) == 1:
+            return result_list[0]
+        else:
+            return result_list
+
+
+@dataclass
+class Permission:
+    name: str
+
+    @staticmethod
+    def resolve_permissions(root: etree.Element) -> list:
+        if root is None:
+            return None
+        permissions = []
+        for permission in root:
+            permissions.append(Permission(permission.find("name").text))
+
+        if len(permissions) == 1:
+            return permissions[0]
+        else:
+            return permissions
+
+
+@dataclass
+class FamilyCount:
+    current: int
+    growing: int
+
+    @staticmethod
+    def resolve_family_count(root: etree.Element) -> "FamilyCount":
+        if root is None:
+            return None
+        return FamilyCount(
+            current=int(root.text), growing=int(root.find("growing").text)
+        )
+
+
+@dataclass
+class NvtCount:
+    current: int
+    growing: int
+
+    @staticmethod
+    def resolve_nvt_count(root: etree.Element) -> "NvtCount":
+        if root is None:
+            return None
+        return NvtCount(
+            current=int(root.text), growing=int(root.find("growing").text)
+        )
+
+
+@dataclass
+class ReportCount:
+    current: int
+    finished: int
+
+    @staticmethod
+    def resolve_report_count(root: etree.Element) -> "ReportCount":
+        if root is None:
+            return None
+        return ReportCount(
+            current=get_int(root.text),
+            finished=get_int_from_element(root, "finished"),
+        )
+
+
+@dataclass
+class Config:
```

Comment:
I would love to use the full name ScanConfig instead of Config
```suggestion
class ScanConfig:
```